selftests/.data/loader_instrumented/imports.py

```diff
@@ -1,4 +1,4 @@
-# docstrings, methods, imports... pylint: disable=C0111,W0403,R0903,C0411,E0611,C0412
+# docstrings, methods, imports... pylint: disable=C0111,R0903,C0411,E0611,C0412
```

Comment:
Here you are removing one warning but the commit message says "Lint E0012 fix". Could, please, elaborate a little bit ?

Reply:
Yes the problem is that there are no imports so disabling the `W0403` doesn't make sense. And the `E0012` is complaining about it. `E0012 checks if a bad value for a PyLint option is encountered.` I understand the confusion and I will update the commit message.